app/models/compute_resources/foreman/model/libvirt.rb

```diff
@@ -230,6 +230,12 @@ def normalize_vm_attrs(vm_attrs)
 
     protected
 
+    def libvirt_connection_error
+      msg = _('Unable to connect to libvirt due to: %s. ') % errors.full_messages.join(', ')
+      msg += _('Please make sure your libvirt compute resource is reachable and that you have appropriate access permissions.')
+      raise Foreman::Exception.new(msg)
```

Comment:
The interpolation must go into exception as a parameter. Also use just one string:

```ruby
msg = N_("Full message)
raise Foreman::Exception.new(msg, errors.full_messages.join(', '))
```